### DISCUSSION TITLE
TYPE=CURSEFORGE error

### DISCUSSION THREAD

**Discussion by Amatsuka-Mao** [itzg/docker-minecraft-server#2960] [Q&A]
**docker command :**
docker run -d  -v /path/on/host:/data -e TYPE=CURSEFORGE \
    -e CF_SERVER_MOD=https://mediafilez.forgecdn.net/files/5502/270/More%20Animals%20%28Server%20Pack%29%20-%20MC%201.21%20-%202.0.0.zip \
    -e MAX_MEMORY=4G \
    -p 25566:25566 -e EULA=TRUE --name mc2 [ghcr.io/itzg/minecraft-server:java21-graalvm](http://ghcr.io/itzg/minecraft-server:java21-graalvm)

**container logs:**
[init] Running as uid=1000 gid=1000 with /data as 'drwxr-xr-x 11 1000 1000 4096 Jul  5 08:43 /data'

[init] WARNING for the old TYPE=CURSEFORGE mechanism it is best to set VERSION to a specific value

[init] Resolved version given LATEST into 1.21

[init] Looking for Feed-The-Beast / CurseForge server modpack.

[init] Downloading modpack from https://mediafilez.forgecdn.net/files/5502/270/More%20Animals%20%28Server%20Pack%29%20-%20MC%201.21%20-%202.0.0.zip...

[init] Unpacking FTB server modpack /data/More Animals (Server Pack) - MC 1.21 - 2.0.0.zip ...

  End-of-central-directory signature not found.  Either this file is not

  a zipfile, or it constitutes one disk of a multi-part archive.  In the

  latter case the central directory and zipfile comment will be found on

  the last disk(s) of this archive.

unzip:  cannot find zipfile directory in one of /data/More Animals (Server Pack) - MC 1.21 - 2.0.0.zip or

        /data/More Animals (Server Pack) - MC 1.21 - 2.0.0.zip.zip, and cannot find /data/More Animals (Server Pack) - MC 1.21 - 2.0.0.zip.ZIP, period.

[init] ERROR: Modpack missing start script and unable to find Forge jar to generate one

**Comment by itzg** [comment]
Looks like the file got corrupted during download. Remove the zip file from your host data directory, re-pull the image to ensure you have the newest, and start the container again.

**Reply by Amatsuka-Mao** [reply]
I can't use the download link from my default browser, but I can use the download link from Firefox, but the package is incomplete on the server.
